packages/react-components/src/components/MessageThread.tsx

```diff
@@ -270,6 +270,13 @@ const DefaultChatMessageRenderer: DefaultMessageRendererType = (props: MessagePr
               : formatTimeForChatMessage(payload.createdOn)
             : undefined
         }
+        style={
+          getRTL()
+            ? payload.mine
```

Comment:
Is this change due to the same northstar issue you mentioned below?
```
 // This is a bug in fluentui react northstar not reversing left and right margins for the author
// in the message bubble
```

If so can you add a similar comment here (as ideally we shouldn't need to have conditional margins styles)